Add capabilities listing with their description (#128)
kyma-project/community: capabilities/display-name.md

---
displayName: Display Name
epicsLabels:
  - area/displayName1
  - area/displayName2
---

> <!-- above metadata will be used on kyma.project.io page to display nice name of capability and have a reference to label that should be used while fetching from ZenHub/GitHub the information about related Epics and their delivery plan   -->

Reviewer: ```suggestion
<!-- The metadata is used on the "https://kyma-project.io/ " page to display a descriptive name of the capability and have a reference to the label that should be used while fetching information about related Epics and their delivery plan from ZenHub and GitHub.   -->
```

> ## Goal

Reviewer 2: To me the difference between "goal" and "vision" is not clear enough as both represents a future target state to achieve. 

I'd replace "goal" with "scope" or "description" to make it clear what is part of the capability and what role it plays in the context of kyma.

Author: O like `scope`

> This is one paragraph that explains what is the scope of the capability. Short paragraph explanation what is the purpose of the capability.

Reviewer: ```suggestion
Include a one-paragraph explanation of the capability's purpose and scope.
```

## Vision

* For best experience 
* It is a bulleted list of elements
* That create a vision

> But it can also be a text if bullet points do not make sense. It can also be a combination of both. Just keep in mind to limit yourself to vision only. This is not a place for specific proposals explanations of features descriptions. This is what Epics are for. Epic if based on proposal, should have proper proposal references.

Reviewer: ```suggestion
It can also be a text if bullet points do not make sense. It can also be a combination of both. Just keep in mind that you should limit yourself to the vision. This is not a place for specific proposals explanations or features descriptions. This is what Epics are for. An Epic is based on a proposal and should have proper proposal references.
```
